src/mmrelay/plugin_loader.py

```
@@ -870,62 +923,543 @@ def _check_auto_install_enabled(config):
+            _run_git(command, cwd=plugins_dir, timeout=120)
+            logger.info(
+                f"Cloned repository {repo_name} from {redacted_url} at {ref_type} {branch_name}"
+            )
+
+            if ref_type != "branch" or not is_default_branch:
+                # Post-clone operations for tags and non-default branches
+                if ref_type == "tag":
+                    _try_fetch_and_checkout_tag(repo_path, ref_value, repo_name)
+                elif ref_type == "branch":
+                    _try_checkout_as_branch(repo_path, ref_value, repo_name)
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)